Formula/zeek.rb

```diff
@@ -42,7 +42,7 @@ def install
     (buildpath/"auxil/c-ares").rmtree
 
     # Remove SDK paths from zeek-config. This breaks usage with other SDKs.
-    # https://github.com/corelight/zeek-community-id/issues/15
+    # https://github.com/Homebrew/homebrew-core/pull/74932
```

Comment:
This PR is also closed.

Reply:
We are able to reference closed PR's in our own repos.

Reply:
👍